indico/web/client/js/custom_elements/ind_with_tooltip.js

```diff
@@ -0,0 +1,50 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import {TipBase} from 'indico/custom_elements/TipBase';
+
+const tipDelay = 1000; // ms
```

Comment:
This feels too slow. What about using something around 100-250ms? (I tested it and it feels way more reactive that way)

Reply:
So there's this:

<img width="485" alt="image" src="https://github.com/indico/indico/assets/65413/17dc98d3-2e21-4144-83f9-bab766e20b8f">

But then again, many authors use about .5s delay instead. 

So not sure what is "typical" really. At any rate, 100~250 is way too low. This delay is used to minimize distraction while tabbing through the interface.

Reply:
OK fair enough. Probably this particular tooltip is not that useful except for very novice Indico users anyway!

(and if we ever move the event creation dialog to react I could absolutely see the button to just become "Create event" and the type selection going in the dialog itself instad of being nested in a dropdown)